frr-reloader/frr-reloader.sh

```diff
@@ -14,14 +14,14 @@ reload_frr() {
   kill_sleep
 
   echo "Checking the configuration file syntax"
-  if ! python3 /usr/lib/frr/frr-reload.py --test --stdout "$FILE_TO_RELOAD" ; then
+  if ! python3 /usr/lib/frr/frr-reload.py --test --stdout "$FILE_TO_RELOAD" 2>&1 | sed 's/password.*/password <retracted>/g'; then
```

Comment:
I believe the if might take into account the return code of the sed and not the PIPESTATUS of the python3 command (I could be wrong!)

Reply:
this is a valid point I think. Thanks for the comment, let me dig more!

Reply:
added set -o pipefail , thanks again for the catch!